src/tike/operators/numpy/convolution.py

```diff
@@ -30,10 +28,10 @@ class Convolution(Operator):
     psi : (ntheta, nz, n) complex64
         The complex wavefront modulation of the object.
     probe : complex64
-        The (ntheta, nscan // fly, fly, nmode, probe_shape, probe_shape)
+        The (ntheta, nscan // fly, fly, 1, probe_shape, probe_shape)
         complex illumination function.
     nearplane: complex64
-        The (ntheta, nscan // fly, fly, nmode, probe_shape, probe_shape)
+        The (ntheta, nscan // fly, fly, 1, probe_shape, probe_shape)
```

Comment:
multi-mode turned off?

Reply:
Yes. We decided months ago that the operator should process only one mode at a time to reduce the memory footprint! This is just some docs I forgot to update.